src/tests/ftest/pool/attribute.py

```diff
@@ -30,7 +30,7 @@
 import threading
 import string
 import random
-from avocado       import Test
+from avocado import Test
 
 sys.path.append('./util')
```

Comment:
Might be orthogonal issue, but this shows up scattered in some tests, not in others; is this necessary uniformly, and if so, should it be moved into apricot as a standardized bit of setup?

Reply:
These would just be in tests that are not yet using apricot, but are in this patch just for the whilespace cleanup, for example.